tools/realsense-viewer/presets/L535 Max Range With Binning.preset

```diff
@@ -1,8 +1,23 @@
 {
-    "Laser Power": 100,
-    "Rx Sensitivity": 1,
-    "Receiver Sensitivity": 9,
-    "Transmitter Frequency": 1,
-    "Vertical Binning": 1,
-    "Min Distance": 1000
+    "device": {
+        "fw version": "04.06.01.09",
+        "name": "Intel RealSense L535",
+        "product line": "L500"
+    },
+    "parameters": {
+        "Laser Power": 100,
+        "Auto Rx Sensitivity": 1,
```

Comment:
How did "Rx Sensitivity" turn into "Auto Rx Sensitivity?

Reply:
it was a bug it should be auto